docs/Agent/installation/ansible_based_installation.md

```diff
@@ -1,16 +1,16 @@
 # Ansible based installation
 
-In the following, we describe how to install Pbench Agent using an ansible playbook.
+In the following: we describe how to install Pbench Agent using an ansible playbook.
```

Comment:
The reference to Ansible here (and the one at line 19) should match the ones at lines 11 and 13.